contribute/guidelines.md

```diff
@@ -56,7 +53,7 @@ Due to a `pip` limitation, it is required to increment the major version number
 Pull requests should generally be **reviewed** by someone else than their authors.
 
 This is mandatory for:
-- Any Pull Request on **`openfisca-core`**
```

Comment:
Why remove this line ? If I read this block, I understand that we are more rigid on `openfisca-france`, `openfisca-web-api`, and `openfisca-parsers`, which is not the case.